_includes/sections/working-here/your-pay-pension-and-other-benefits.md

```diff
@@ -108,38 +108,100 @@ your holiday allowance shown on your BreatheHR dashboard.
 
 #### Sickness
 
-If you are sick, you must let us know by 10:00, or as soon as is reasonably
-practical. Do not come to the office. If you come to the office when sick, you
-may be sent home again.
-
-If you're sick but able to work, you can work from home. Everyone at dxw should
-have a laptop, and should generally take it home with them in case it's needed.
-If you're sick and at home but don't have your laptop, it can be couriered to
-you.
-
-When you return to work, you must check that your sickness has been correctly
-recorded in BreatheHR, and update it if necessary. If you're sick for more than
-a week, you'll also need to provide a
-[fit note](https://www.gov.uk/government/publications/the-fit-note-a-guide-for-patients-and-employees).
-
-If you're sick on more than 10 occasions or for more than 10 days over a
-12-month period, we'll invite you to an absence meeting to discuss your
-situation. We may agree with you an action plan and/or a review period. We may
-seek medical advice. Should your attendance fail to improve after a three-month
-period, we will invite you to a further absence meeting to discuss your
-situation. In this meeting we will also give consideration to your possible
-dismissal.
-
-We'll also invite you to an absence meeting if you're off sick for more than
-three weeks on a single occasion. In addition to the above, we will seek to
-support your return to work and discuss with you any temporary or permanent
-adjustments which may assist you. If you are unable to return to work in your
-current role we will consider if there is a suitable alternative role. If we are
-unable to reach an agreement that will permit you to return to work, we will
-give consideration to your possible dismissal.
-
-We will always only consider dismissal after we have exhausted all other
-reasonable options.
+This policy covers both short and longer term sickness absences. Our aim is to
+look after the health and wellbeing of all dxw staff and the needs of the
+company.
+
+##### Communication of sickness
+
+If you’re sick, you must let your line manager or a member of the HR team know
+by 10:00am or as soon as you reasonably can after that. This can be done via
+email, directly on Slack, phone call or text. Line managers will record and
+monitor sickness for individuals within their teams.
```

Comment:
Do the line managers need to tell anyone? How should they do that?

Reply:
Line managers are responsible for updating breathehr but unless specifically asked by the individual there is no responsibility to tell anyone in the wider team

Reply:
That feels like a gap. How is a DL supposed to know why someone isn't here. In the case of a team without a DL, how is the client supposed to know? What if they're on support and need cover?

Reply:
If the person sick is informing their line manager they can also ask this to be passed on to the wider team, the DL or a direct message can be sent. At the moment not everyone uses the sickness slack channel and will reach out to those needing to know directly.

Reply:
So the person who's sick needs to check their calendar and let their line manager know everyone they might need to tell? That seems pretty inefficient and a toll on the sick person, who should be resting, not worrying about who needs to know that they're sick.

Reply:
No - letting your line manager or HR know directly if you have anyone (inclu the client or delivery lead) to be told will suffice. BreatheHR would also be updated with your sickness if someone reaches out to HR on a need to know basis. Dependent on the role or client work a team member is on this will change.

Reply:
Right, but to work out that list of people, I'd need to check my calendar.